programmers2/짝수와-홀수.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    if(num%2 == 0) {
+        return "Even";
+    }
+    else {
+        return "Odd";
+    }
```

Comment:
공백 컨벤션에 조금 더 신경쓸것
js 에서 비교할때는 == 말고 ===을 사용함.
 ===0 을 생략하고 if()가 1일때 나오는 출력값 even, odd 를 반대로 해서 더 간결하게 가능하다.
 ```suggestion
    if (num % 2)  return "Odd"; // return 이 한줄일때 이와같이 중괄호 생략가능
    else  return "Even";
```